docs/mkdocs.yml

```diff
@@ -89,15 +89,25 @@ theme:
       primary: black
       accent: blue
 
-extra_css:
-  - stylesheets/extra.css
-
 repo_url: https://github.com/JetBrains/koog
 repo_name: "Koog on GitHub"
 edit_uri: edit/develop/docs/docs/
 copyright: Copyright © 2000-2025 JetBrains s.r.o.
 
+extra_css:
+  - stylesheets/extra.css
+
 extra:
+  analytics:
+    provider: google
+    property: GTM-5P98
+  consent:
+    title: Cookie consent
+    description: >- 
```

Comment:
Was this message copied from our websites?

Reply:
I've asked for assistance from web team here, as I'm not sure where to get the right one for us